### ISSUE TITLE
Extract Activity lifecycle method names to constants for Robolectric's tests

### ISSUE THREAD

**Issue by utzcoz** [robolectric/robolectric#6941]
### Description

The tests of Robolectric use pure strings for `Activity` lifecycle method names at many places, for example https://github.com/robolectric/robolectric/search?q=%22onCreate%22. Those names are fixed, and we can extract them to constants for tests to avoid writing them again and again for different tests.

### Steps to Reproduce

None.

### Robolectric & Android Version

Current master branch.

### Link to a public git repo demonstrating the problem:

Robolectric itself.

**Comment by arinmodi** [comment]
@utzcoz, I want to work on this,

So I have to just create one string resource and replace the fixed lifecycle method names with resource, right ?

**Comment by arinmodi** [comment]
> @utzcoz, I want to work on this,
> 
> So I have to just create one string resource and replace the fixed lifecycle method names with resource, right ?

@utzcoz  ?

**Comment by utzcoz** [comment]
@arinmodi Yep. You can create a util class and extract these values to constants in this util class. This utils class can be created into test directory instead of released src directory, and I hope every test module has a separate util class for this purpose if many modules need this refactor.
